src/util/event-to-promise.test.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env jest */
```

Comment:
To test the eventToPromise function, you can extend your [EventEmitter](https://nodejs.org/api/events.html#events_class_eventemitter) mock with a way to emit events.

Then you can expect that the returned promise is resolved/rejected after you have emitted certain events.

Adding test(s) for the filter options would be nice if you have time for it. Otherwise add it as TODO comment.